crates/bitwarden-crypto/Cargo.toml

```diff
@@ -48,6 +50,7 @@ sha2 = ">=0.10.6, <0.11"
 subtle = ">=2.5.0, <3.0"
 thiserror = { workspace = true }
 tsify-next = { workspace = true, optional = true }
+typenum = ">=1.18.0, <1.19.0"
```

Comment:
In other areas we get `typenum` using `generic_array` or `aes`. I'm not against depending on it directly but we probably want to update all consumers to use it directly?

Reply:
Updated all references to typenum.